capa/engine.py

```diff
@@ -79,6 +80,9 @@ def __init__(self, children, description=None):
         self.children = children
 
     def evaluate(self, ctx):
+        capa.perf.counters["evaluate.feature"] += 1
+        capa.perf.counters["evaluate.feature.and"] += 1
```

Comment:
this will have a marginal performance impact itself. we can potentially remove these once our performance work is complete.